tests/test_planning.py

```diff
@@ -21,8 +22,16 @@ def test_sesame_plan():
     nsrts = get_gt_nsrts(env.predicates, env.options)
     task = next(env.train_tasks_generator())[0]
     option_model = create_option_model(CFG.option_model_name, env.simulate)
-    plan = sesame_plan(task, option_model, nsrts, env.predicates, 1, 123)
+    plan, _ = sesame_plan(task, option_model, nsrts, env.predicates,
```

Comment:
I'm confused, was this a bug in the old unit test?

Reply:
yes lol the metrics return wasn't being considered